internaltools/cryptogen/ca/ca.go

```diff
@@ -1,344 +0,0 @@
-/*
```

Comment:
You could have moved this file and made edits. At least in the review, it shows as deleted. Hence, it added difficulties to the review process.

Reply:
Sorry. I should have made this in two steps.